apps/previewgenerator.sh

```diff
@@ -110,6 +112,10 @@ then
     clear      
 
 else
+    # remove imagick
+    apt purge php-imagick -y
```

Comment:
This is not needed as it's not installed in the VM. Good thinking though.

Reply:
if you run the script a second time it could happen though.

Reply:
Then please check if the package exists first. Thanks.

Reply:
Have added the check.
So should be done.